### PR TITLE
Refactor out grid resizing JS

### PR DESCRIPTION
**Pull request by ace-n**
As title

### CODE REVIEW

**Comment by ace-n** [comment]
Assigning @achalv to sign off - @colegleason you're welcome to do this too.

File: liquid/static/css/sponsors.css [hunk 1]
```diff
@@ -14,5 +19,5 @@
 .company-card img {
   margin: 0 auto;
   display: block;
-  max-height: 100%;
+  max-height: 46px; /* For oversized logos; TODO ask companies for consistent sizes */
```
Comment (colegleason) [inline liquid/static/css/sponsors.css]: Instead of asking for a certain size we should just rescale. If not in CSS, we can do it on image upload.

**Comment by colegleason** [comment]
Can I get a screenshot here?

File: liquid/static/css/sponsors.css (hunk 1 above)
Comment (ace-n) [inline liquid/static/css/sponsors.css]: Yeah - this is a hotfix.

**Comment by ace-n** [comment]
screen shot 2015-08-01 at 6 05 36 pm

(N.B: Since my copy of the DB is out of date, I disabled JS on the live-site page and added my CSS changes to get this image.)

**Comment by colegleason** [comment]
What happens when we get enough sponsors to span multiple lines?

What was the JS solving that is now not needed?

**Comment by ace-n** [comment]
Sponsors on multiple lines:
screen shot 2015-08-08 at 5 02 02 pm

Masonry is (according to its site) a library for doing grid layouts on pages with rectangles of different, dynamic sizes. Since all the logos *should* be the same size (and are pretty close to it right now), we don't need to have such a library in use.

**Comment by colegleason** [comment]
Ok, LGTM then.

**Pull request merged**